docs/resource-sharing/os-backfill-containers.md

```diff
@@ -74,93 +82,71 @@ docker run -it --rm --user osg  \
        opensciencegrid/osgvo-docker-pilot:release
 ```
 
-Replace `/path/to/token` with the location you saved the token obtained from the OSP administrators.
+Replace `/path/to/token` with the location you saved the token obtained from the OSPool Token Registry.
```

Comment:
Thanks for getting rid of "OSP". Now I won't have to fight down my "grr fix typo" reflexes :-)